Gemfile

```diff
@@ -3,10 +3,11 @@ source "https://rubygems.org"
 gem "rails"
 
 # database and caching
+
+gem 'mysql2'
```

Comment:
This change is why the build is failing. You'll need to revert it before I can merge.